release-content/0.14/release-notes/12792_Implement_Auto_Exposure_plugin.md

```diff
@@ -1,4 +1,13 @@
-<!-- Implement Auto Exposure plugin -->
-<!-- https://github.com/bevyengine/bevy/pull/12792 -->
+Since **Bevy 0.13**, you can configure the the EV100 of a camera, which allows you to adjust the exposure of the camera in a physically based way. This also allows you to dynamically change the exposure values for various effects. However, this is a manual process and requires you to adjust the exposure values yourself.
 
-<!-- TODO -->
+**Bevy 0.14** introduces **Auto Exposure**, which automatically adjusts the exposure of your scene based on the brightness of the scene. This can be useful when you want to create the feeling of a very high dynamic range, since your eyes also adjust to large changes in brightness. Note that this is not a replacement for hand-tuning the exposure values, rather an additional tool that you can use to create dramatic effects when brightness changes rapidly.
```

Comment:
```suggestion
**Bevy 0.14** introduces **Auto Exposure**, which automatically adjusts the exposure of your camera based on the brightness of the scene. This can be useful when you want to create the feeling of a very high dynamic range, since your eyes also adjust to large changes in brightness. Note that this is not a replacement for hand-tuning the exposure values, rather an additional tool that you can use to create dramatic effects when brightness changes rapidly.
```